### DISCUSSION TITLE
Autoplay?

### DISCUSSION THREAD

**Discussion by diego-francisco-amador** [ciderapp/Cider#1147] [Q&A] [answered]
Not sure If I'm overlooking the option, but is there a way to enable Autoplay like in Apple Music web? I would like music to play "endlessly" in the background.

**Comment by KillerParticles** [comment, marked as answer]
Yes! Its the same as AM web; open the Queue and toggle this button here:
![image](https://user-images.githubusercontent.com/93286201/172885902-89f897c6-9444-4bc2-8140-ec4745eb92a5.png)

**Reply by diego-francisco-amador** [reply]
Got it, thanks! I was not seeing it before when I had the Apple Music Web theme enabled. I changed it back to the default theme and I see the button now. Appreciate the quick response.

Before
![image](https://user-images.githubusercontent.com/57582224/172887432-8318d710-a9b2-4045-9469-77df34dea649.png)

After
![image](https://user-images.githubusercontent.com/57582224/172887798-f76c2de9-dc4d-45c9-946a-7adc0936bb2a.png)
